app/components/HeroSection.tsx

```diff
@@ -0,0 +1,18 @@
+import heroImg from "../imgs/Hero-Image.png"
+
+export default function HeroSection() {
```

Comment:
Personal preference: Let's avoid default exports and instead export everything as none default. That way, we can easily export several components from the same file!

Reply:
It sounds great to me!